### ISSUE TITLE
bug: EventBridge Scheduled Rules generate an empty event

### ISSUE THREAD

**Issue by ph-l** [localstack/localstack#5387]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When a scheduled rules runs, an empty event is generated and sent to targets.

### Expected Behavior

An event like this sample should be sent.
```json
{
  "version": "0",
  "id": "89d1a02d-5ec7-412e-82f5-13505f849b41",
  "detail-type": "Scheduled Event",
  "source": "aws.events",
  "account": "123456789012",
  "time": "2016-12-30T18:44:49Z",
  "region": "us-east-1",
  "resources": ["arn:aws:events:us-east-1:123456789012:rule/SampleRule"],
  "detail": {}
}
```

### How are you starting LocalStack?

With a docker-compose file

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

    docker-compose up

#### Client commands (e.g., AWS SDK code snippet, or sequence of "awslocal" commands)

    awslocal events put-rule --name scheduled_rule --schedule-expression "rate(1 minutes)"
    awslocal events put-targets --rule scheduled_rule --targets "Id"="1","Arn"="arn:aws:logs:us-east-1:000000000000:log-group:/aws/events/echo"
    awslocal--endpoint-url=http://localhost:4566 logs tail "/aws/events/echo"

### Environment

```markdown
- OS: macOS Monterey
- LocalStack: 0.13.3
```

### Anything else?
I think [this](https://github.com/localstack/localstack/blob/1567f25a8f672f4dec23cb7cb1aadb5a744366f8/localstack/services/events/events_listener.py#L92) means an non-empty event is sent only when an input is `Constant (JSON text)`.
![image](https://user-images.githubusercontent.com/4127030/151643813-a4d1e113-dba3-41ad-8400-da57e4f674a1.png)

**Comment by caiosantesso** [comment]
I second this one. 

Docker container version:
```
root@6ce6e86c0347:~# /opt/code/localstack/bin/localstack --version
0.14.1
```

Commands and outputs:
```
$ awslocal logs create-log-group --log-group-name "/aws/events/echo"
$ awslocal events put-rule --name scheduled_rule --schedule-expression "rate(1 minutes)"
$ awslocal events put-targets --rule scheduled_rule --targets "Id"="1","Arn"="arn:aws:logs:us-east-1:000000000000:log-group:/aws/events/echo"
$ awslocal logs describe-log-streams --log-group-name /aws/events/echo
{
    "logStreams": [
        {
            "logStreamName": "31ccea0e-9861-4926-bdbd-8f2d34623406",
            "creationTime": 1647201075202,
            "firstEventTimestamp": 1647201075205,
            "lastEventTimestamp": 1647201075205,
            "lastIngestionTime": 1647201075211,
            "uploadSequenceToken": "1",
            "arn": "arn:aws:logs:us-east-1:000000000000:log-group:/aws/events/echo:log-stream:31ccea0e-9861-4926-bdbd-8f2d34623406",
            "storedBytes": 2
        }
    ]
}
$ awslocal logs get-log-events --log-group-name /aws/events/echo --log-stream-name 31ccea0e-9861-4926-bdbd-8f2d34623406
{
    "events": [
        {
            "timestamp": 1647201075205,
            "message": "{}",
            "ingestionTime": 1647201075211
        }
    ],
    "nextForwardToken": "f/00000000000000000000000000000000000000000000000000000000",
    "nextBackwardToken": "b/00000000000000000000000000000000000000000000000000000000"
}
```

**Comment by alexrashed** [comment]
Hi @ph-l!
I think this issue should have been fixed a while ago with https://github.com/localstack/localstack/pull/6124. /cc @lukqw @dominikschubert 
Could you please verify if this issue is still persistent in the latest version of LocalStack?
Thanks!

**Comment by ph-l** [comment]
Hi @alexrashed,

I don't think it's fixed yet.
```
$ docker-compose up
[+] Running 1/0
 ✔ Container localstack_main  Created                                                                                                                                                                                                     0.0s 
Attaching to localstack_main
localstack_main  | 
localstack_main  | LocalStack version: 2.2.1.dev
localstack_main  | 
localstack_main  | LocalStack Docker container id: 0989e4091f48
localstack_main  | LocalStack build date: 2023-09-13
localstack_main  | LocalStack build git hash: fa10d1d7
```

```
$ awslocal logs create-log-group --log-group-name "/aws/events/echo"
$ awslocal events put-rule --name scheduled_rule --schedule-expression "rate(1 minutes)"
{
    "RuleArn": "arn:aws:events:us-east-1:000000000000:rule/scheduled_rule"
}
$ awslocal events put-targets --rule scheduled_rule --targets "Id"="1","Arn"="arn:aws:logs:us-east-1:000000000000:log-group:/aws/events/echo"
{
    "FailedEntryCount": 0,
    "FailedEntries": []
}
$ awslocal logs describe-log-streams --log-group-name /aws/events/echo
{
    "logStreams": [
        {
            "logStreamName": "232363e9-b1cc-48df-91c0-7a5c4dd69f22",
            "creationTime": 1694655112549,
            "firstEventTimestamp": 1694655112551,
            "lastEventTimestamp": 1694655112551,
            "lastIngestionTime": 1694655112555,
            "uploadSequenceToken": "1",
            "arn": "arn:aws:logs:us-east-1:000000000000:log-group:/aws/events/echo:log-stream:232363e9-b1cc-48df-91c0-7a5c4dd69f22",
            "storedBytes": 2
        }
    ]
}
$ awslocal logs get-log-events --log-group-name /aws/events/echo --log-stream-name
{
    "events": [
        {
            "timestamp": 1694655112551,
            "message": "{}",
            "ingestionTime": 1694655112555
        }
    ],
    "nextForwardToken": "f/00000000000000000000000000000000000000000000000000000000",
    "nextBackwardToken": "b/00000000000000000000000000000000000000000000000000000000"
}
```

**Comment by thrau** [comment]
I can confirm this issue is still present in 2.3, there's even a TODO in the code for scheduled events: https://github.com/localstack/localstack/blob/e7cf994ab6edf29d585d13699e6306722d03145a/localstack/services/events/provider.py#L112

i'm currently writing some tests for this and will add a resolution as soon as those are in place.

**Comment by thrau** [comment]
Default events for scheduled rules should now work. We have tests for sqs and logs.

**Comment by ph-l** [comment]
@thrau Should this be closed? I tested with version 2.3.2 and it appears EventBridge is still sending an empty event.

**Comment by alexrashed** [comment]
@ph-l The PR which fixed this issue (which was merged two weeks ago) is not contained in `2.3.2` (which was released 3 weeks ago), but only in `latest` for now. Could you please try using the `latest` tag of the docker image?

**Comment by ph-l** [comment]
Sorry for my mistake. Just tested with `latest` and looks good now.
